docker-fetch.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+set -e
+
+test -d dl || \
+        mkdir dl
+test -f dl/vlfeat_K80L3.SIFT.tree || \
+        wget https://gitlab.com/alicevision/trainedVocabularyTreeData/raw/master/vlfeat_K80L3.SIFT.tree -O dl/vlfeat_K80L3.SIFT.tree
+test -f dl/cmake-3.16.3.tar.gz || \
+        wget https://cmake.org/files/v3.16/cmake-3.16.3.tar.gz -O dl/cmake-3.16.3.tar.gz
+test -d  dl/deps || \
+	mkdir dl/deps
+
+test -d build-fetch || {
+	mkdir -p build-fetch/external
+	ln -s ../../dl/deps build-fetch/external/download
+}
+
+pushd build-fetch
+cmake .. \
+     -DCMAKE_BUILD_TYPE=Release \
+     -DALICEVISION_BUILD_DEPENDENCIES:BOOL=ON \
+     -DAV_BUILD_ALICEVISION:BOOL=OFF
+
+cmake -P "opencv-prefix/src/opencv-stamp/download-opencv.cmake"
```

Comment:
Why do you manually call the download step?

Reply:
This script is called in the host environment, before the docker build steps, to download as many dependencies as possible to avoid downloading the same content repeatedly for each supported configuration.